requirements-prod.in

```diff
@@ -0,0 +1,2 @@
+uwsgi<3
+factory_boy
```

Comment:
I would think that in production, we do not require `factory_boy`. Though it seems to also be in `requirements.in` so we don't need to include it here.

Reply:
Thanks, I removed it now from the prod requirements.